server/datastore/mysql/maintained_apps.go

```diff
@@ -128,6 +129,10 @@ WHERE NOT EXISTS (
 	)
 )`
 
+	if opt.MatchQuery != "" {
+		stmt += fmt.Sprintf(`AND (fla.name LIKE '%%%s%%')`, opt.MatchQuery)
```

Comment:
can you use this function that'll do what you're doing here but more safely? https://github.com/fleetdm/fleet/blob/d9bb0ea466430daef3a2541ec94754f2dd708132/server/datastore/mysql/mysql.go#L1134-L1138

Reply:
updated to use this method